build.sbt

```diff
@@ -60,7 +60,7 @@ lazy val client = (project in file("client"))
   .dependsOn(sharedJs)
   .settings(commonSettings: _*)
   .settings(
-    persistLauncher := true,
+    persistLauncher := false,
```

Comment:
Disable auto start of JsApp, because I see different scripts on different forms, so that I want to control which script should be executed at which html page